src/py/flwr/client/app.py

```diff
@@ -491,14 +492,10 @@ def _load_client_app() -> ClientApp:
                     # Load ClientApp instance
                     client_app: ClientApp = load_client_app_fn()
 
+                    # Execute ClientApp
                     reply_message = client_app(message=message, context=context)
-                    # Update node state
-                    node_state.update_context(
-                        run_id=message.metadata.run_id,
-                        context=context,
-                    )
                 except Exception as ex:  # pylint: disable=broad-exception-caught
-                    log(ERROR, "ClientApp raised an exception", exc_info=ex)
+                    log(ERROR, "SuperNode raised an exception", exc_info=ex)
```

Comment:
Isn't the common case here `ClientApp` raising an exception?
```suggestion
                    log(ERROR, "ClientApp raised an exception", exc_info=ex)
```

Reply:
It's the most likely scenario but since we are also having the `load_client_app_fn()` inside the `try`, that can potentially trigger its own exception.

Reply:
should me make the message conditional based on the type of exception we get?